lago/plugins/vm.py

```diff
@@ -268,17 +284,13 @@ def _extract_paths_scp(self, paths, ignore_nopath):
                     remote_path=host_path,
                     propagate_fail=False
                 )
+            except LagoCopyFilesFromVMError as err:
+                LOGGER.debug('%s: LagoCopyFilesFromVMError', err.args[0])
             except SCPException as err:
-                err_substr = ': No such file or directory'
-                if len(err.args) > 0 and isinstance(
-                    err.args[0], basestring
-                ) and err_substr in err.args[0]:
-                    if ignore_nopath:
-                        LOGGER.debug('%s: ignoring', err.args[0])
-                    else:
-                        raise ExtractPathNoPathError(err.args[0])
+                if ignore_nopath:
```

Comment:
This `if` block should be under the first except, because it's the one which handles files the doesn't exist.

Reply:
If I move the block to the previous except it will fails.
It provides an exception on:
SCPException: scp: /var/log/something.log: No such file or directory
The ignore path in this case is related to the scp how to perform